src/components/RightPanel.tsx

```diff
@@ -12,6 +12,12 @@ const StickySearch = styled.div`
   top: 1.5%;
 `;
 
+const FixMenu = styled.div`
+  position: fixed;
```

Comment:
I hope it won't hurt to scroll the right panel when there are a lot of recommendations